index.html

```diff
@@ -944,8 +1028,9 @@ <h2>Services</h2>
 
             <p>
 [=Services=] are used in [=controller documents=] to express ways of
-communicating with the [=subject=] or associated entities. A [=service=] can be
-any type of service the [=subject=] wants to advertise for further discovery,
+communicating with the [=controller=], or associated entities, in relation to 
```

Comment:
```suggestion
communicating with the [=documentController=], or associated entities, in relation to 
```

Reply:
-1 for the reasons stated here: https://github.com/w3c/controller-document/pull/116/files#r1839014775

Reply:
I'm marking all of @David-Chadwick's document controller / proof controller suggestions as resolved as they are now in a new issue #119